styles/_base.scss

```diff
@@ -286,22 +291,43 @@ ul, ol {
 
 #milestone {
   // delete this rule in production
-  position: fixed;
-  bottom: -4px;
-  padding: 4px 132px 22px 8px;
-  font-size: 14px;
-  right: -6px;
-  border-radius: 4px;
-  border-width: 2px;
-  border-style: solid;
-  z-index: 10;
-
-  .milestone-inner {
-    position: relative;
-    top: 10px;
+  display: none;
+
+  .pre-release & {
+    display: block;
```

Comment:
Adjusts the milestone callout on the bottom to not overlap the chat sidebar, and only be visible when the pre-release class is present. This makes it simpler to test the layout for release by turning that class off.